packages/client/src/components/SafeSettings/components/Assets/AssetTableView.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { useClipboard } from "hooks";
+import { formatAddress, getFlowscanUrlForContract } from "utils";
+import { Copy } from "components/Svg";
+
+const AssetTableView = ({ assets, emptyPlaceholder, onRemoveClick }) => {
+    const clipboard = useClipboard();
+
+    return (
+        <div className="border-light rounded-sm py-3 table-border">
+            {
+                assets.length > 0 ?
+                    assets.map(asset => {
+                        const address = asset?.split(".")[1];
+                        const name = asset?.split(".")[2];
+                        const formattedAddress = formatAddress(address);
+
+                        return (
+                            <div className="mx-0 p-3 columns" key={name}>
+                                <div className="column">{name}</div>
+                                <div className="column has-text-weight-bold">
+                                    {formattedAddress}
+                                    <span className="pointer" onClick={() => clipboard.copy(formattedAddress)}>
+                                        <Copy className="mt-1 ml-2 pointer" />
+                                    </span></div>
+                                <a
+                                    className="column is-flex is-justify-content-end has-text-purple button border-none"
```

Comment:
This a tag doesn't have any child, what are we trying to achieve for giving it flex class and justify content end?

Reply:
By adding is-flex and is-justify-content-end this button is centered with the other text from the table and aligned to the right, same like the Remove button.

Reply:
I don't think that's how they should be used. {display:flex} and {justify-content:flex-end} have impact on the children dom not the container itself, refer to this guide: https://css-tricks.com/snippets/css/a-guide-to-flexbox/
To achieve the effect you described, attach the flex classes to the parent dom (lin 19) instead

Reply:
Instead of using is-flex, I removed the is-flex, justify content and column classes, the 'column' class was messing with the position of the button. It's properly aligned now.